core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -356,7 +357,7 @@ static struct tee_mmap_region *init_xlation_table(struct tee_mmap_region *mm,
 
 	assert(level <= 3);
 
-	debug_print("New xlat table (level %u):", level);
+	debug_print("New xlat table (level %u) [0x%8x]:", level, cur);
```

Comment:
oups. bug. That a copy paste from non-lpae. I understand I did not test the LPAE with the MMU debug traces enabled.
I will fix.